test/generator/exercise_case_test.rb

```diff
@@ -3,45 +3,41 @@
 module Generator
   class ExerciseCaseTest < Minitest::Test
     def test_name
-      assert_equal 'test_foo', ExerciseCase.new(description: 'foo').name
+      subject = ExerciseCase.new(canonical: OpenStruct.new('description' => 'foo'))
```

Comment:
Why the change in style from `symbol: key` to `'string' => key` here, while maintaining the style of `symbol: key` in the most of the rest of the file?  (see line 34 as another deviation from the style)